en/clubs/dkm/sidebar.md

```diff
@@ -7,7 +7,8 @@ Complaints, compliments, gifts?</br>
 **DKM President (Klubbmästare):**  [klubbm@datasektionen.se](mailto:klubbm@datasektionen.se)
 
 Assortment and other bar-related stuff?</br>
-**DKM Bar responsible (Barmästare):** [barm@d.kth.se](mailto:barm@d.kth.se)
+**DKM Bar responsible (Barmästare):** [barm@d.kth.se](mailto:barm@datasektionen.se)
```

Comment:
Was about to comment that this email did not exist last I checked, but it seems to have been added now. Nice! 🎉